src/top.rs

```diff
@@ -130,6 +150,27 @@ impl<'a> Iterator for LinkIter<'a> {
     }
 }
 
+#[derive(Debug, Copy, Clone, Hash, Eq, PartialEq, Ord, PartialOrd)]
```

Comment:
Would be nice to have some docs here

Reply:
Wonderful feedback. I just realized we don't need this type.